src/settings.rs

```diff
@@ -2,54 +2,30 @@
 use std::{net::SocketAddr, path::PathBuf};
 
 use anyhow::Result;
-use config::{builder::DefaultState, Config as cfg, ConfigBuilder, ConfigError, File};
+use config::{builder::DefaultState, Config as cfg, ConfigBuilder, ConfigError, File, FileFormat};
 use serde::{de::Error, Deserialize, Deserializer};
 
+use crate::CmdLineArgs;
+
 const DEFAULT_GIGANTO_NAME: &str = "localhost";
 const DEFAULT_GIGANTO_INGEST_SRV_ADDR: &str = "[::]:38370";
 const DEFAULT_GIGANTO_PUBLISH_SRV_ADDR: &str = "[::]:38371";
-const DEFAULT_REVIEW_NAME: &str = "localhost";
-const DEFAULT_REVIEW_RPC_SRV_ADDR: &str = "[::]:38390";
+const DEFAULT_LAST_TIMESTAMP_DATA_PATH: &str = "/usr/local/aice/conf/time_data.json";
```

Comment:
I believe there are a few considerations regarding `DEFAULT_LAST_TIMESTAMP_DATA_PATH`:

- `Settings::last_timestamp_data` relates to this issue: https://github.com/aicers/crusher/issues/13.
  - Saving the start timestamp of a timeseries to a file seems to be a feature that Crusher must consistently execute during normal operation.
- At an organizational level, we are trying to avoid specifying default paths for items in configuration files that are path-based.

Considering these factors, I would like to ask not to define `DEFAULT_LAST_TIMESTAMP_DATA_PATH`.

If Crusher is running in local configuration mode and the `last_timestamp_data` path provided in the local configuration file is invalid, I think Crusher needs to be terminated, and inform the users to provide an appropriate path for the `last_timestamp_data` configuration item.

Reply:
Removed the const.
Now, it shows an error message as below if `last_timestamp_data` is not provided.

> Error: failed to parse configuration file
>
>Caused by:
 >   missing field `last_timestamp_data`